src/main/java/pib/gui/MainWindow.java

```diff
@@ -0,0 +1,54 @@
+package pib.gui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import pib.Pib;
+
+/**
+ * Controller for pib.gui.MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Pib pib;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image pibImage = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
+
+    @FXML
+    public void initialize() {
+        scrollPane.vvalueProperty().bind(dialogContainer.heightProperty());
+    }
+
+    public void setPib(Pib p) {
+        pib = p;
+        dialogContainer.getChildren().add(DialogBox.getPibDialog(pib.loadData(), pibImage));
+    }
```

Comment:
Small nit but perhaps you could add some Javadocs here too since they are public. Noticed the same issue in a few other places too.